src/motor_node.cc

```diff
@@ -143,6 +146,12 @@ void initMcbParameters(std::unique_ptr<MotorHardware> &robot )
         } else if (g_node_params.wheel_type == "thin"){
             wheel_type = MotorMessage::OPT_WHEEL_TYPE_THIN;
             ROS_INFO("Host is specifying wheel_type of '%s'", "thin");
+
+            // If thin wheels and no drive_type is in yaml file we will use 4wd
+            if (g_node_params.drive_type == "firmware_default") {
```

Comment:
The better option would be to go into the 4wd repo and make sure the drive type is defined as 4wd there, but assuming nobody else is using thin wheels right now I guess we can leave it at that.